widget/embedded/src/index.ts

```diff
@@ -40,7 +40,8 @@ import {
 import { useWallets } from '@rango-dev/wallets-react';
 
 import { WidgetWallets } from './Wallets';
```

Comment:
This is not related to your changes, but could you move `./Wallets` to `./cointaners/Wallets` as well?

Reply:
fixed.